webrtc/RTCPeerConnection-addTrack.https.html

```diff
@@ -64,6 +64,62 @@
               transceiver be the result.
           4.  Add transceiver to connection's set of transceivers.
    */
+  function addTrackFromGetUserMedia(t) {
```

Comment:
might be better to call `setup()`, `setupTest()` etc

Reply:
And to use async function and await getUserMedia?

Reply:
done locally.